templates/includes/sidebar.html

```diff
@@ -0,0 +1,14 @@
+<ul class="sidebar">
+  <li>
+    <a href="#">Home page</a>
+  </li>
+  <li>
+    <a href="#">Manufacturers</a>
+  </li>
+  <li>
+    <a href="#">Cars</a>
+  </li>
+  <li>
+    <a href="#">Drivers</a>
```

Comment:
The href attributes for the links are currently set to '#', which means they are placeholders and not functional. Consider updating these links to point to the correct URLs for the Home page, Manufacturers, Cars, and Drivers sections.